rootfs/etc/init.d/atlantis.sh

```diff
@@ -82,11 +82,19 @@ if [ "${ATLANTIS_ENABLED}" == "true" ]; then
 	# Do not export these as Terraform environment variables
 	export TFENV_BLACKLIST="^(AWS_ACCESS_KEY_ID|AWS_SECRET_ACCESS_KEY|AWS_SECURITY_TOKEN|AWS_SESSION_TOKEN|ATLANTIS_.*|GITHUB_.*)$"
 
-    # Configure Git credentials for atlantis to allow access to GitHub private repos (see rootfs/usr/local/bin/git-credential-github)
-    # https://git-scm.com/book/en/v2/Git-Tools-Credential-Storage
+    # Configure git credentials for atlantis to allow access to GitHub private repos
     export GITHUB_USER=${ATLANTIS_GH_USER}
     export GITHUB_TOKEN=${ATLANTIS_GH_TOKEN}
     chmod +x /usr/local/bin/git-credential-github
+
+    # Force git to use HTTPS instead of SSH. With HTTPS, git will use the `GITHUB_TOKEN` to authenticate with GitHub (with SSH it won't)
+    # https://ricostacruz.com/til/github-always-ssh
+    # https://git-scm.com/docs/git-config#Documentation/git-config.txt-urlltbasegtinsteadOf
+    # https://gist.github.com/Kovrinic/ea5e7123ab5c97d451804ea222ecd78a
+    git config --global url."https://github.com/".insteadOf git@github.com:
```

Comment:
Quotes are not needed as in original example?

Reply:
added quotes
but they have examples with quotes and without on the same page
https://ricostacruz.com/til/github-always-ssh
https://gist.github.com/Kovrinic/ea5e7123ab5c97d451804ea222ecd78a

(maybe just `git://github` does not require quotes)